test/gui/tst_syncing/test.feature

```diff
@@ -353,4 +353,15 @@ Feature: Syncing files
         Then as "Alice" folder "folder1" should exist on the server
         And as user "Alice" folder "folder1" should contain "500" items on the server
         And as "Alice" folder "folder2" should exist on the server
-        And as user "Alice" folder "folder2" should contain "500" items on the server
\ No newline at end of file
+        And as user "Alice" folder "folder2" should contain "500" items on the server
+
+
+    Scenario: Default options in advanced configuration
```

Comment:
the scenario should open up/ give an idea about what scenario tests.
I think it can be better